Lil-Peep/src/styles/modules/site-header.scss

```diff
@@ -250,7 +250,7 @@
     }
 
     .header-logo {
-      margin-bottom: 0;
+      margin: 00Xpx;
```

Comment:
This should be written as `margin: 0;` no need for the units (px). 

P.S. The _X_ I put on my previous comment was a representation that any numeric value could be put there, but I think I failed. :sweat_smile: